src/Language/PureScript/CoreFn/CSE.hs

```diff
@@ -216,7 +216,7 @@ newScope isTopLevel body = local goDeeper $ do
     if isTopLevel
     then env{ _depth = depth', _deepestTopLevelScope = depth' }
     else env{ _depth = depth' }
-    where 
+    where
```

Comment:
Is this change caused by whitespace? If so, can that be undone?

Reply:
Yeah there was a trailing whitespace my editor deleted, I can undo that if you want to keep it

Reply:
I say leave it; ordinarily I'd want to keep changes like this from distracting from the diff but this diff is tiny and the change is very unobjectionable.